site/learn/expert-advice/authentication/webauthn-explained.md

```diff
@@ -0,0 +1,515 @@
+
+However, the user experience is suboptimal. It can be a hassle to force someone to go to a different device to get a code.
+
+Depending on the method, it can also be insecure. For example, SMS codes can be phished. If you are using push messaging, attackers can bombard users until [MFA fatigue sets in](https://techcommunity.microsoft.com/t5/microsoft-entra-azure-ad-blog/defend-your-users-from-mfa-fatigue-attacks/ba-p/2365677).
+
+### Single Sign-on
+
+Single sign-on is another option, which doesn't have these same risks. To be more precise, it centralizes them. This makes sense, as most people pay more attention to their Google/Facebook/Apple account than to the many random accounts they've created on other sites. You can depend on a big company's engineers to encourage best practices.
+
+However, single sign-on has a business risk. You are now depending on a third party to allow your users to access your website.
+
+## WebAuthn
+
+WebAuthn is a decentralized, consumer-friendly, secure alternative for protecting your clownwear store.
+
+With WebAuthn, the private keys which authenticate the user to the website typically remain on the authenticator. (There are some services that sync them across devices, but that is not part of the specification.) The server stores only public keys, which means that, unlike password hashes, there's not much value in obtaining them.
+
+WebAuthn is phishing-resistant because of built-in browser security checks. These include:
+
+* Each credential is locked to a specific domain name and won't be presented to any other domain
+* WebAuthn won't work if the requesting site is not on HTTPS
+
+Additionally, WebAuthn has additional security benefits:
+
+* Physical access to the authenticator is required
+* Unlike a code, a user can't share their passkey, because they don't know it
+* Unlike a push message, attackers can't start a WebAuthn ceremony
+
+These all combine to offer a more secure experience than MFA or a password manager. 
+
+The specifics of the authentication method are dictated by the authenticator. Here are some examples of authentication experiences that a user can have during a WebAuthn authentication ceremony:
+
+* facial recognition
+* PIN entry
+* fingerprint recognition
+
+### Browser Support
+
+WebAuthn is available on all modern browsers except for IE and Opera Mini. Sorry, IE and Opera Mini users!
+
+{% include _image.liquid src="/assets/img/advice/webauthn-explained/can-i-use-webauthn.png" alt="Screen explaining browser support for WebAuthn." class="img-fluid" figure=false %}
+
+### The Authentication Ceremony, Revisited
+
+You've already seen the high-level authentication flow, but let's take a closer look. When a user logs in with WebAuthn, a typical flow is:
+
+1. The user signals they want to use WebAuthn. This could be a previously set cookie, clicking a button, or some other manner.
```

Comment:
A cookie may serve as an indicator that the user should be prompted to use WebAuthn, but the ceremony should be triggered by a user action. Safari is very picky about this and will log a warning that WebAuthn was called without user interaction when you trigger the WebAuthn ceremony on page load with JavaScript.